instance_control/readme.md

```diff
@@ -28,15 +37,15 @@ bubuku_cluster.py terminate \
 ```
 
 ### Attach
-
+The command creates a new instance and attaches provided volume to that instance.
```

Comment:
I even did it in the first iteration, but removed in favour of the manual control of volumes you want to attach, because it is possible that the step of setting tag(Name for example) for the volume can fail and you have to figure out what happened. I see a lot of improvements foe this scripts, but let give it some time to work in production.